### ISSUE TITLE
libargon2.so.1 not found while executing php

### ISSUE THREAD

**Issue by kerunaru** [Homebrew/linuxbrew-core#14132]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [ ] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)
- [ ] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [X] ran `brew update` and can still reproduce the problem?
- [X] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [X] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

```
$ brew config

HOMEBREW_VERSION: 2.1.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 2b83463091513826127c14acae81a7c354dfce69
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: 526a2022904f220f539987749e53c1c2d44e83e2
Core tap last commit: 15 hours ago
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CACHE: /home/kerunaru/.cache/Homebrew
HOMEBREW_LOGS: /home/kerunaru/.cache/Homebrew/Logs
CPU: octa-core 64-bit skylake
Homebrew Ruby: 2.3.7 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/bin/ruby
Clang: N/A
Git: 2.22.0 => /home/linuxbrew/.linuxbrew/bin/git
Curl: 7.58.0 => /usr/bin/curl
Kernel: Linux 4.15.0-54-generic x86_64 GNU/Linux
OS: elementary OS 5.0 Juno (juno)
Host glibc: 2.27
/usr/bin/gcc: 7.4.0
glibc: N/A
gcc: N/A
xorg: 20170115_1

$ brew doctor

Your system is ready to brew.
```

To help us debug your issue please explain:

- What you were trying to do (and why)

  - I'm trying to execute `php` from a terminal window.

- What happened (include command output)

  - PHP isn't finding libargon2.so.1 and fails its execution.

`php: error while loading shared libraries: libargon2.so.1: cannot open shared object file: No such file or directory`

- What you expected to happen

  - PHP executes correctly.

- Step-by-step reproduction instructions (by running `brew install` commands)

```
$ brew update
$ brew install php
$ php
```

**Comment by kerunaru** [comment]
I workaround this with:

```
$ cd /home/linuxbrew/.linuxbrew/opt/argon2/lib
$ ln -s x86_64-linux-gnu/libargon2.so.1 libargon2.so.1
```

**Comment by iMichka** [comment]
I confirm that the bug is also happening on our CI: #14141

**Comment by iMichka** [comment]
Looks like this is a known bug: https://www.linuxquestions.org/questions/showthread.php?p=6012100

There is a patch that has been proposed, I will test this.

**Comment by iMichka** [comment]
Fixed by #14255
